curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,527 @@ Which of the following is the default value of the `position` property?
+
+#### --answer--
+
+It positions the element relative to its nearest positioned ancestor.
+
+### --question--
+
+#### --text--
+
+What is the difference between `position: fixed` and `position: sticky`?
+
+#### --distractors--
+
+`fixed` positions the element relative to its parent, while `sticky` positions it relative to the viewport.
+
+---
+
+`fixed` keeps the element in the normal flow, while `sticky` removes it.
+
+---
+
+`fixed` allows scrolling, while `sticky` does not.
+
+#### --answer--
+
+`fixed` positions the element relative to the viewport, while `sticky` positions it relative to its nearest scrollable ancestor.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `z-index`?
+
+#### --distractors--
+
+```css
+.box {
+  position: static;
+  z-index: 10;
+}
+```
+
+---
+
+```css
+.box {
+  position: relative;
+}
+```
+
+---
+
+```css
+.box {
+  position: absolute;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: absolute;
+  z-index: 5;
+}
+```
+
+### --question--
+
+#### --text--
+
+What is the default stacking context in CSS?
+
+#### --distractors--
+
+The `body` element.
+
+---
+
+The `html` element.
+
+---
+
+The `viewport`.
+
+#### --answer--
+
+The root element of the document.
+
+### --question--
+
+#### --text--
+
+Which of the following is true about `position: sticky`?
+
+#### --distractors--
+
+It positions the element relative to the viewport.
+
+---
+
+It removes the element from the normal flow.
+
+---
+
+It keeps the element fixed at all times.
+
+#### --answer--
+
+It positions the element relative to its nearest scrollable ancestor and becomes fixed when scrolling past a threshold.
+
+### --question--
+
+#### --text--
+
+What is the effect of `position: relative` on an element?
+
+#### --distractors--
+
+It removes the element from the normal flow.
+
+---
+
+It positions the element relative to the viewport.
+
+---
+
+It keeps the element fixed during scrolling.
+
+#### --answer--
+
+It offsets the element from its normal position without removing it from the document flow.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `position: fixed`?
+
+#### --distractors--
+
+```css
+.box {
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+```
+
+---
+
+```css
+.box {
+  position: absolute;
+  top: 20px;
+  left: 20px;
+}
+```
+
+---
+
+```css
+.box {
+  position: static;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+```
+
+### --question--
+
+#### --text--
+
+What is the purpose of `position: absolute`?
+
+#### --distractors--
+
+To keep the element in the normal flow.
+
+---
+
+To position the element relative to the viewport.
+
+---
+
+To make the element sticky.
+
+#### --answer--
+
+To position the element relative to its nearest positioned ancestor.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `position: sticky`?
```

Comment:
this should also be updated because all of the wrong answers use something other than sticky.
this make it to easy to pick the right one.
this need to be updated to something else.